pkg/apis/migration.harvesterhci.io/v1beta1/virtualmachines.go

```diff
@@ -92,4 +92,5 @@ const (
 	VirtualMachineImageFailed     condition.Cond = "VirtualMachineImageFailed"
 	VirtualMachineExportFailed    condition.Cond = "VMExportFailed"
 	VirtualMachineMigrationFailed ImportStatus   = "VMMigrationFailed"
+	SkipPreflightChecks           string         = "harvesterhci.io/skip-preflight-checks"
```

Comment:
Could you explain why you prefer using an annotation over adding a new field to `VirtualMachineImportSpec`?

Reply:
good point, i have moved it to a field in the spec.